.github/monorepo/split-repositories.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+set -e -o pipefail
+
+SPLIT_BRANCH=$1
+REMOTE_TEMPLATE=$2
+FORCE=${3:-false}
+
+set -u
+
+# Import functions
+. $(dirname "$0")/monorepo_functions.sh
+
+assert_split_branch_variable
+assert_remote_template_variable
+
+if [[ "$FORCE" == true ]]; then
+    assert_split_branch_is_not_protected
```

Comment:
We use to force split for example 12.0 during release. This will disable it, right?

Reply:
It will not be necessary anymore to force split those versions as they will be split automatically